### DISCUSSION TITLE
Product Increment 11 Iteration 3

### DISCUSSION THREAD

**Discussion by jzbmw** [eclipse-tractusx/item-relationship-service#314] [General]
## Minister Security
@ds-psosnowski 
Timebox: 2h
- [ ] Check Veracode for findings
- [ ] Check Invict for findings
- [ ] run and chack Dashtool IP

## Release Champion: 
@dsmf 
Timebox: 2h

- [ ] Has the Release been conducted (Release is being skipped!)
- [ ] Check for Dependabots PR's on Tractus-X

## Docu-Guru:
@ds-jhartmann 
Timebox: 2h 
- [ ] Focused on Documentation of Tickets

## Testmanager
@ds-alexander-bulgakov 
- [ ] Update [Release 24.03 ---- Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.03+----+Content+Overview)
- [ ] Update Test Status - PI11 [Test Status - PI11](https://confluence.catena-x.net/display/PL/Test+Status+-+PI11)
- [ ] Update [Integration Test - PI11 - Release 2024.03](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI11+-+Release+2024.03)

## Firefighter
@mkanal 
- [ ] Maintenance of our community
- [ ] Response to Github issues https://github.com/eclipse-tractusx/item-relationship-service/issues

## Demonstrator:  
@ds-jhartmann 
- [x] Conduct the PI Systemdemo

**Comment by ds-alexander-bulgakov** [comment]
Testmanager update:
- No updated necessary for [Release 24.03 Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.03+----+Content+Overview)
- [Test Status - PI11](https://confluence.catena-x.net/display/PL/Test+Status+-+PI11) is updated for cucumber tests. Tavern tests will be updated once the ESS-INT is running with testdata. Currently tests fail because ESS-testdata is not available as expected. **Update** (13th dec): execution ticket of tavern tests has been added. ESS-testdata is available.
- [Integration Test - PI11 - Release 2024.03](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI11+-+Release+2024.03) has been updated with released version 4.3.0 on 8th of Dec.
